sw/device/tests/autogen/meson.build

```diff
@@ -0,0 +1,48 @@
+# Copyright lowRISC contributors.
+# Licensed under the Apache License, Version 2.0, see LICENSE for details.
+# SPDX-License-Identifier: Apache-2.0
+
+###############################################################################
+# IP Integration Tests
+###############################################################################
+plic_all_irqs_test_lib = declare_dependency(
+  link_with: static_library(
+    'plic_all_irqs_test_lib',
+    sources: ['plic_all_irqs_test.c'],
+    dependencies: [
+      # sw_lib_dif_adc_ctrl,
```

Comment:
Is the expectation that someone will be manually switching these as they become necessary, or that a script will regenerate this file? Seems like the latter. Probably worth marking all generated files as such so people don't get confused.

Reply:
Kinda confused about what the comments are achieving if this isn't human editable. Humans should never look at these files, so why put comments in them?

Reply:
Right but... the file will get regenerated, right? Like I think we should not be emitting these comments at all.

Reply:
Don't really care either way. Just want this PR merged today, so, whatever it takes. Removed.